src/components/Register/Register.jsx

```diff
@@ -0,0 +1,178 @@
+import React from "react";
+import { Form, Field } from "react-final-form";
+import { useNavigate, Link } from "react-router-dom";
+import "./Register.scss";
```

Comment:
```suggestion
import "./register.scss";
```
the name of css files should be in lower case

Reply:
done